python/cuspatial/cuspatial/core/binops/contains.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+from cudf import Series
+from cudf.core.column import as_column
+
+from cuspatial._lib.pairwise_point_in_polygon import (
+    pairwise_point_in_polygon as cpp_pairwise_point_in_polygon,
+)
+from cuspatial._lib.point_in_polygon import (
+    point_in_polygon as cpp_point_in_polygon,
+)
+from cuspatial.utils.column_utils import normalize_point_columns
+
+
+def contains_properly(
+    test_points_x,
+    test_points_y,
+    poly_offsets,
+    poly_ring_offsets,
+    poly_points_x,
+    poly_points_y,
+):
+    """Compute from a series of points and a series of polygons which points
+    fall within each polygon. Note that polygons must be closed: the first and
+    last coordinate of each polygon must be the same.
+
+    This implements `.contains_properly`, which shares a large
+    space of correct cases with `GeoPandas.contains` but they do not
+    produce identical results. In the future we will use intersection
+    testing to match .contains behavior.
```

Comment:
Please don't use "we" or discuss future plans.

And I think we should learn from the shapely documentation here, it is nice.  https://shapely.readthedocs.io/en/latest/reference/shapely.contains_properly.html

No need to discuss spaces of correct cases. :)

```suggestion
    """Compute from a series of points and a series of polygons which points
    are properly contained within the corresponding polygon. Polygon A contains Point B 
    properly if B intersects the interior of A but not the boundary (or exterior). 
    
    Note that polygons must be closed: the first and last vertex of each
    polygon must be the same.
```